integrations/slack/webhooks/bot_events.go

```diff
@@ -43,8 +43,8 @@ var BotEventHandlers = map[string]BotEventHandler{
 	// TODO: im_history_changed
 
 	"member_joined_channel": events.ChannelGroupHandler,
-
-	"message": events.MessageHandler,
+	"member_left_channel":   events.MemberLeftChannelHandler,
+	"message":               events.MessageHandler,
```

Comment:
Keep a separator empty line between "member" events and "message" events.

```suggestion
	"member_left_channel":   events.MemberLeftChannelHandler,

	"message": events.MessageHandler,
```